### PR TITLE
changes in createcontact and two more file

### PR DESCRIPTION
**Pull request by priyankasandbhor95**
changes in readdatafromproperties
and changes in createcontact

### CODE REVIEW

File: com.practice.DDT/src/test/java/com/practice/ddtproperty/readprpertyfiledata.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class readprpertyfiledata {
 
 	public static void main(String[] args) throws IOException {
 		// TODO Auto-generated method stub
-		
+		//code
```
Comment (atharvasandbhor03) [inline com.practice.DDT/src/test/java/com/practice/ddtproperty/readprpertyfiledata.java:18]: add more comments 1

File: com.practice.gen/src/test/java/Modulegeneric_Contact_test/createContact_tc_01Test.java [hunk 2]
```diff
@@ -24,6 +24,7 @@ public class createContact_tc_01Test {
 	public static void main(String[] args) throws IOException {
 		// TODO Auto-generated method stub
 		// TODO Auto-generated method stub
+		//code
```
Comment (atharvasandbhor03) [inline com.practice.gen/src/test/java/Modulegeneric_Contact_test/createContact_tc_01Test.java:27]: add more comments 2

**Review by atharvasandbhor03 — COMMENTED** [review]
do changes asap

File: com.practice.gen/src/test/java/Modulegeneric_Contact_test/createContact_tc_01Test.java (hunk 2 above)
Comment (priyankasandbhor95) [reply com.practice.gen/src/test/java/Modulegeneric_Contact_test/createContact_tc_01Test.java:27]: all task done

**Comment by atharvasandbhor03** [comment]
merged

**Pull request merged**